packages/react-dom/package.json

```diff
@@ -40,12 +41,14 @@
     "unstable_testing.js",
     "unstable_server-external-runtime.js",
     "cjs/",
-    "umd/"
+    "umd/",
+    "esm/"
```

Comment:
I think this is technically no longer needed since the bun builds now use the cjs output

Reply:
Removed!